util/lang/Editors.java

```diff
@@ -306,7 +306,7 @@ else if (t.mult == 0)
 
 		map().put("WAVE", new EditControl<>(Proc.WAVE.class, (t) -> {
 			t.prob = MathUtil.clip(t.prob, 0, 100);
-			t.lv = MathUtil.clip(t.lv, 1, 20);
+			t.lv = t.lv != 0 ? t.lv : 1;
```

Comment:
In line 309, `MathUtil.clip` is for clipping the value, basically this means `Math.max(1, Math.min(t.lv, 20))`. Due to this change, clipping will be removed, and users will be able to put level something like 100. Please clip the level